src/main/kotlin/com/hedvig/accountservice/graphql/resolvers/Query.kt

```diff
@@ -74,18 +74,19 @@ class Query(
     }
 
     private fun getEstimatedSubscription(memberId: String): MonetaryAmount {
-        val now = YearMonth.now()
-        val currentProductSubscriptions = productPricingService.getProductSubscriptions(memberId, now)
+        val currentProductSubscriptions = productPricingService.getProductSubscriptions(memberId, getMonthForEstimation())
```

Comment:
I'm personally a fan of putting variables outside function calls i.e. 
```suggestion
        val monthForEstimation = getMonthForEstimation()
        val currentProductSubscriptions = productPricingService.getProductSubscriptions(memberId, monthForEstimation)
```
to make the code clear. This is just an opinion though so it's nothing worth fixing.